modules/ocf/manifests/apt.pp

```diff
@@ -92,6 +92,19 @@
       notify => Exec['apt_update'];
     }
 
+    package { 'pkg-mozilla-archive-keyring':; }
+
+    apt::source {
+      'mozilla':
+        location => 'http://mozilla.debian.net/',
+        release  => "${::lsbdistcodename}-backports",
+        repos    => 'firefox-release',
```

Comment:
Agreed. We're currently using the old ESR. The new ESR was [released a few days ago](https://www.mozilla.org/en-US/firefox/45.0esr/releasenotes/) and is based on Firefox 45, the latest release.